fe/free_energy_rabfe.py

```diff
@@ -61,6 +61,41 @@ def from_log(cls, log_line):
         values = list(map(float, value_strings))
         return RABFEResult(mol_name, *values)
 
+    @classmethod
+    def _convert_field_to_sdf_field(cls, field_name: str) -> str:
+        cleaned_name = field_name.replace("_", " ")
+        return cleaned_name
+
+    @classmethod
+    def from_mol(cls, mol):
+        field_names = fields(cls)
+
+        kwargs = {}
+        for field in field_names:
+            if field.name == "mol_name":
+                field_name = "_Name"
+                val = mol.GetProp(field_name)
+            else:
+                field_name = cls._convert_field_to_sdf_field(field.name)
+                val = float(mol.GetProp(field_name))
+            kwargs[field.name] = val
+        return RABFEResult(**kwargs)
+
+    def apply_to_mol(self, mol: Chem.Mol):
+        results_dict = asdict(self)
+        results_dict.update(
+            {
+                "dG_bind": self.dG_bind,
+                "dG_bind_err": self.dG_bind_err,
+            }
+        )
+        for field, val in results_dict.items():
+            if field == "mol_name":
+                field_name = "_Name"
+            else:
+                field_name = self._convert_field_to_sdf_field(field)
```

Comment:
Would move this branch into `_convert_field_to_sdf_field`

Reply:
Nice -- resolved in https://github.com/proteneer/timemachine/pull/566/commits/7d9255f508f552def69a751680bf86b170d73504